DeUrgenta/Features/EmergencyBackpack/View/MyBackpackView.swift

```diff
@@ -0,0 +1,79 @@
+import SwiftUI
+
+struct MyBackpackView: View {
+    let items = [
+        CategoryItem(title: AppStrings.CategoryItem.foodTitle.localized(), icon: "icon-syrup", type: .food),
```

Comment:
We should have the hardcoded categories in a ViewModel - there we will replace them later maybe with an API request.